hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/upgrade/EightToSevenDowngradeHandler.java

```diff
@@ -32,6 +39,26 @@
 public class EightToSevenDowngradeHandler implements DowngradeHandler {
   @Override
   public Map<ConfigProperty, String> downgrade(HoodieWriteConfig config, HoodieEngineContext context, String instantTime, SupportsUpgradeDowngrade upgradeDowngradeHelper) {
+    HoodieTableMetaClient metaClient = HoodieTableMetaClient.builder().setConf(context.getStorageConf().newInstance()).setBasePath(config.getBasePath()).build();
+    List<HoodieInstant> instants = metaClient.getActiveTimeline().getInstants();
+    if (!instants.isEmpty()) {
+      context.map(instants, instant -> {
+        if (!instant.getFileName().contains("_")) {
+          return false;
+        }
+        try {
+          StoragePath fromPath = new StoragePath(metaClient.getMetaPath(), instant.getFileName());
```

Comment:
We need to add a todo here when we bring separate action name for clustering/compaction in 1.x as well.